pkgs/development/libraries/gstreamer/good/extra-modules.sh

```diff
@@ -0,0 +1,8 @@
+addGIOModulesLibPath () {
+    if [[ -d "$1/lib/gio/modules" && "$1" == *glib-networking* ]]
```

Comment:
```suggestion
    if [[ -d "$1/lib/gio/modules" && -n "$(ls -A "$1/lib/gio/modules")" ]]
```
should be more like [find_gio_modules](https://github.com/NixOS/nixpkgs/blob/ffa859d8048388564984368f5f88b84e34fe178b/pkgs/build-support/setup-hooks/wrap-gapps-hook/wrap-gapps-hook.sh#L4) and could also remove duplicates.

this is my first time making a setup hook and this is basically a copy of [gstreamer/core/setup-hook.sh](https://github.com/NixOS/nixpkgs/blob/ffa859d8048388564984368f5f88b84e34fe178b/pkgs/development/libraries/gstreamer/core/setup-hook.sh)